docsrc/imap/download/release-notes/index.rst

```diff
@@ -128,23 +118,24 @@ Development snapshots
 Older Versions
 ==============
 
-Series 1
---------
+Series 2: 2.0 - 2.5
+-------------------
 
 .. toctree::
     :maxdepth: 1
 
-    1/1.x.x
+    2.5/index
+    2.4-dav/index
+    2.4/index
+    2.3/index
+    2.2/2.2.x
+    2.1/2.1.x
+    2.0/2.0.x
```

Comment:
Do any of these pre-2.5 links go anywhere useful?  2.5 was the first version that we used RST for the docs, 2.4 and earlier just had a directory of html files that used to get published statically on a server that doesn't exist anymore, and I don't know/remember what happens for them anymore.